services/uws-api-server/values-summit.yaml

```diff
@@ -0,0 +1,55 @@
+uws-api-server:
+  targetCluster: "summit"
+  hostname: summit-lsp.lsst.codes
+  image:
+    tag: v1.0.0
+  workingVolume:
+    name: job-files
+    server: nfs1.cp.lsst.org
+    exportPath: "/scratch"
+    claimName: uws-server-pvc
+    mountPath: "/uws"
+    subPath: "uws"
+  volumes:
+  - name: project
+    server: nfs1.cp.lsst.org
+    claimName: project-pvc
+    mountPath: "/project"
+    exportPath: "/project"
+    subPath: ""
+    readOnly: false
+  - name: home
+    server: nfs1.cp.lsst.org
+    claimName: home-pvc
+    mountPath: "/jhome"
+    exportPath: "/jhome"
+    subPath: ""
+    readOnly: false
```

Comment:
I'm not sure `/jhome` should be available to OCPS/UWS.  If we have pipelines or data taken from (or put) there, we will have a hard time with reproducibility.